docs/documentation/full-text/filtering.mdx

```diff
@@ -2,96 +2,83 @@
+|                                            | `int2`            | `int4[]`           |
+|                                            | `int2`            | `int8[]`           |
+|                                            | `int4`            | `int8[]`           |
+|                                            | `float4`          | `float4[]`         |
+|                                            | `float8`          | `float8[]`         |
+|                                            | `float4`          | `float8[]`         |
+|                                            | `date`            | `date[]`           |
+|                                            | `time`            | `time[]`           |
+|                                            | `timetz`          | `timetz[]`         |
+|                                            | `timestamp`       | `timestamp[]`      |
+|                                            | `timestamptz`     | `timestamptz[]`    |
+|                                            | `text`            | `text[]`           |
+|                                            | `uuid`            | `uuid[]`           |
+| `IS`, `IS NOT`                             | `bool`            | `bool`             | `WHERE in_stock IS true`   |
+| `IS NULL`, `IS NOT NULL`                   | `bool`            |                    | `WHERE rating IS NOT NULL` |
+|                                            | `int2`            |                    |
+|                                            | `int4`            |                    |
+|                                            | `int8`            |                    |
+|                                            | `int2`            |                    |
+|                                            | `int2`            |                    |
+|                                            | `int4`            |                    |
+|                                            | `float4`          |                    |
+|                                            | `float8`          |                    |
+|                                            | `float4`          |                    |
+|                                            | `date`            |                    |
+|                                            | `time`            |                    |
+|                                            | `timetz`          |                    |
+|                                            | `timestamp`       |                    |
+|                                            | `timestamptz`     |                    |
+|                                            | `text`            |                    |
+|                                            | `uuid`            |                    |
+
+As part of our roadmap, we are adding support for more operators and types. To request support for a new type or operator,
```

Comment:
I'm not sure I'd want to encourage that.

Reply:
Really, I don't know that the docs should reference our "roadmap" at all.  The docs define what we currently do/support.